packages/xo-web/src/common/intl/messages.js

```diff
@@ -571,8 +571,9 @@ const messages = {
   newSrPasswordPlaceHolder: 'Password',
   newSrLvmDevicePlaceHolder: 'Device, e.g /dev/sda…',
   newSrLocalPathPlaceHolder: '/path/to/directory',
-  newSrUseNfs4: 'Use NFSv4',
+  newSrNfsDefaultVersion: 'Default NFS version',
   newSrNfsOptions: 'Comma delimited NFS options',
+  newSrUseNfs: 'Use NFS',
```

Comment:
'NFS' ?

Reply:
This is the NFS Input Label, @julien-f what do you think ? 
I'm okey with @Rajaa-BARHTAOUI

Reply:
If's the label for the version selector it should probably be `NFS version` or something similar.

Reply:
And the key should probably reflect this as well :wink